config/root.rc.in

```diff
@@ -0,0 +1,48 @@
+#include <winver.h>
+
+ROOT_ICON   ICON   "../icons/RootIcon.ico"
+
+#define VER_FILEVERSION             @ROOT_MAJOR_VERSION@,@ROOT_MINOR_VERSION@,@ROOT_PATCH_VERSION@,0
+#define VER_FILEVERSION_STR         "@ROOT_MAJOR_VERSION@.@ROOT_MINOR_VERSION@.@ROOT_PATCH_VERSION@.0\0"
```

Comment:
Is the double `\0` really needed? One from you, one from the string literal termination?

Reply:
Taken from Microsoft example...